### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by houmark**
The README is outdated and has 404 links and shows builds not passing. Updated to working links, added link to contributing guide. More should be added here, but this one at least has working links, badges etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-[![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/)
+# Podio PHP client library
 
-This the official PHP Client for interacting with the Podio API. All parts of the Podio API are covered in this client. See [podio.github.io/podio-php](http://podio.github.io/podio-php) for documentation.
+This the non-official PHP Client for interacting with the Podio API maintained by the Podio community. Most parts of the Podio API are covered in this client. See [podio-community.github.io/podio-php](https://podio-community.github.io/podio-php/) for documentation.
 
-[![Build Status](https://travis-ci.org/podio/podio-php.svg?branch=4.0.0)](https://travis-ci.org/podio/podio-php)
+To contribute, please read: [the contribution guide](https://github.com/podio-community/podio-php/blob/master/CONTRIBUTING.md).
+
+[![Build Status](https://travis-ci.org/podio-community/podio-php.svg?branch=4.0.0)](https://travis-ci.org/podio-community/podio-php)
```
Comment (daniel-sc) [inline README.md]:
```suggestion
[![Build Status](https://travis-ci.org/podio-community/podio-php.svg?branch=4.0.0)](https://travis-ci.org/podio-community/podio-php)
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
-[![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/)
+# Podio PHP client library
 
-This the official PHP Client for interacting with the Podio API. All parts of the Podio API are covered in this client. See [podio.github.io/podio-php](http://podio.github.io/podio-php) for documentation.
+This the non-official PHP Client for interacting with the Podio API maintained by the Podio community. Most parts of the Podio API are covered in this client. See [podio-community.github.io/podio-php](https://podio-community.github.io/podio-php/) for documentation.
```
Comment (daniel-sc) [inline README.md]:
```suggestion
This the non-official PHP Client for interacting with the Podio API maintained by the Podio community and the continuation of the former official Podio PHP Client. Most parts of the Podio API are covered in this client. See [podio-community.github.io/podio-php](https://podio-community.github.io/podio-php/) for documentation.
```

**Comment by daniel-sc** [comment]
Hi @houmark 
thanks for the contribution! This looks fine - if you like, please add my suggested changes and then we can merge.

**Comment by houmark** [comment]
All done, this should be ready to merge 👍

**Pull request merged**